### PR TITLE
Added test target Ubuntu 2204 with Psycopg 3

### PR DESCRIPTION
**Pull request by elpavel**
##### SUMMARY
Added a new test target with Ubuntu 2204 and Psycopg 3.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
The test environment

##### ADDITIONAL INFORMATION
Only one new test target for now, to enable automated testing of modules modified to work with the new Psycopg 3 in #499

This PR doesn't touch the collection functionality, it only updates the test environment.

Also updated the test container as per https://github.com/ansible-collections/news-for-maintainers/issues/49

And updated the Remote devel RHEL test to use RHEL 8.8

### CODE REVIEW

**Comment by elpavel** [comment]
The **Remote devel RHEL 8.7** test keeps failing with
```
Bootstrapping Python 3.9
Error: Unable to find a match: python3.9-devel python3.9-pip python3.9-cryptography
Failed to install packages. Sleeping before trying again...
Error: Unable to find a match: python3.9-devel python3.9-pip python3.9-cryptography
Failed to install packages. Sleeping before trying again...
```
no idea why.

File: .azure-pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -134,6 +134,8 @@ stages:
               test: fedora37
             - name: Ubuntu 20.04
               test: ubuntu2004
+            - name: Ubuntu 22.04
```
Comment (Andersson007) [inline .azure-pipelines/azure-pipelines.yml:137]:
```suggestion
            - name: Ubuntu 22.04
```
Should we also add this to test against `- stage: Docker_devel` ?

**Review by Andersson007 — COMMENTED** [review]
@elpavel thanks!

**Review by hunleyd — APPROVED** [review]

**Comment by Andersson007** [comment]
@elpavel thanks for the contribution!
@hunleyd thanks for reviewing! I think two approvals are enough for such PRs, so next time feel free to merge after approval

**Comment by hunleyd** [comment]
@Andersson007 i didn't merge it because, at the time i reviewed it, you had an open discussion

**Comment by Andersson007** [comment]
> @Andersson007 i didn't merge it because, at the time i reviewed it, you had an open discussion

Ah, clear then:) thanks

**Pull request merged**